projects/Mallard/src/screens/settings-screen.tsx

```diff
@@ -140,12 +142,21 @@ const SettingsScreen = ({ navigation }: NavigationInjectedProps) => {
             signedIn: data => [
                 {
                     key: `Sign out`,
-                    title: `Sign out of ${data.userDetails.publicFields.displayName}`,
+                    title: data.userDetails.publicFields.displayName,
                     data: {
                         onPress: async () => {
                             await signOut()
                         },
                     },
+                    proxy: <Text style={{color: '#41A9E0', ...getFont('sans', 1),}}>Sign Out</Text>,
+                },
+                {
+                    key: `Subscription details`,
+                    title: `Subscription details`,
+                    data: {
+                        onPress: async () => {},
```

Comment:
@RichieAHB will integrate into what you're doing to show subscription details here on another screen.